app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/UpdateVatSubscription.scala

```diff
@@ -46,7 +46,8 @@ object UpdateVatSubscription {
 
 }
 
-case class ControlInformation(source: String = "100",
+case class ControlInformation(welshIndicator: Boolean = false,
```

Comment:
I wouldn't give the indicator a default value here as we should always retrieve the correct one before the POST request. The reason the other two are defaulted is because it will always be the same for every user